fastdupes.py

```diff
@@ -439,14 +439,14 @@ def pruneUI(dupeList, mainPos=1, mainLen=1):
     :rtype: :class:`~__builtins__.int`
     """
     dupeList = sorted(dupeList)
-    print
+    print('')
     for pos, val in enumerate(dupeList):
-        print "%d) %s" % (pos + 1, val)
+        print("{} {}".format(pos + 1, val))
     while True:
-        choice = raw_input("[%s/%s] Keepers: " % (mainPos, mainLen)).strip()
```

Comment:
Since you're making Python 2.x-incompatible changes, please also switch the shebang to say `python3` rather than `python`.

(The use of `python` rather than `python2` is just an artifact of this not having been worked on in a while.)